README.md

```diff
@@ -6,8 +6,18 @@ A list of friendly active repos for contributing!
 
```

Comment:
I think the cleanest way to add these would be as part of this list. Can you make that change?

Reply:
Done sir